### DISCUSSION TITLE
The Oily Lads

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#430] [Announcements]
![grafik](https://user-images.githubusercontent.com/16674258/153157109-47ec17e8-f6a2-4269-b920-ad73961cd109.png)

h

**Comment by HbmMods** [comment]
(Doesn't include chemplant recipes like reclaimed, petroil and leaded gasoline)

**Comment by Adam29Adam29** [comment]
:reimumunch:

**Comment by HbmMods** [comment]
:trollface:

**Comment by ncr-t** [comment]
does crack oil yield more ? or is it just so you can cut the recipe a bit ?

**Reply by ncr-t** [reply]
(please explain me I have stupid)

**Reply by HbmMods** [reply]
Crack oil yields crack naphtha and light oil (the more useful bits) while replacing the heavier components (heavy oils which are only good for solid fuel) with aromatic and unsaturated hydrocarbons (i.e. the loopy bois and the really angry lads)

**Reply by ncr-t** [reply]
ohh, thanks
